server/impl/src/main/java/com/walmartlabs/concord/server/org/secret/SecretManager.java

```diff
@@ -547,9 +552,9 @@ private UUID create(DSLContext tx,
         SecretType type = secretType(s);
 
         byte[] pwd = getPwd(password);
-        byte[] salt = secretCfg.getSecretStoreSalt();
+        byte[] salt = SecretUtils.generateSalt(16);
```

Comment:
`16` should be moved into a constant, e.g. `SALT_LENGTH`.

Reply:
Added it as constant